### PR TITLE
CVE-2017-5651

### PR DESCRIPTION
**Pull request by hmeinertrita**
completed CVE-2017-5651 YAML file for round 1 submission

### CODE REVIEW

File: cves/CVE-2017-5651.yml [hunk 1]
```diff
@@ -43,7 +43,13 @@ description_instructions: |
   that outsiders to Chromium would not understand. Technology like "regular
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
-description:
+description: |
+  In certain versions of Apache Tomcat, the refactoring of the HTTP
+  connectors introduced a regression in send file processing. If the processing
```
Comment (mll8657) [inline cves/CVE-2017-5651.yml]: "the refactoring of the HTTP connectors introduced a regression in send file processing" could be rewritten in easier to understand language.

**Comment by mll8657** [comment]
+1 upvotes

File: cves/CVE-2017-5651.yml [hunk 2]
```diff
@@ -4,33 +4,33 @@ nickname_instructions: |
   Nickname is optional. Provide a useful, professional, and catchy nickname for
   this vulnerability. Ideally fewer than 30 characters. This will be shown
   alongside its CVE to make it more easily distinguished from the rest.
-nickname:
+nickname: double processor-caching
```
Comment (moiseslorap) [inline cves/CVE-2017-5651.yml]: Nickname could be less technical.

File: cves/CVE-2017-5651.yml (hunk 1 above)
Comment (moiseslorap) [reply cves/CVE-2017-5651.yml]: I second this

File: cves/CVE-2017-5651.yml [hunk 3]
```diff
@@ -67,7 +73,7 @@ fixes:
      note:
 vccs:
   - commit:
```
Comment (moiseslorap) [inline cves/CVE-2017-5651.yml]: Add commit hash

File: cves/CVE-2017-5651.yml [hunk 4]
```diff
@@ -204,4 +227,20 @@ mistakes:
     Use those questions to inspire your answer. Don't feel obligated to answer
     every one. Write a thoughtful entry here that those ing the software
     engineering industry would find interesting.
-  answer:
+  answer: |
```
Comment (moiseslorap) [inline cves/CVE-2017-5651.yml:229]: How do you think this type of vulnerability can be avoided in the future?

**Review by moiseslorap — COMMENTED** [review]
+3 upvotes

**Pull request closed without merging**